recipes/r-causalimpact/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set version = '1.3.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-causalimpact
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/CausalImpact_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/CausalImpact/CausalImpact_{{ version }}.tar.gz
+  sha256: 9d9fdb1dfd3aae4625bb73ba29a4e587beb0ccdccd6d2d0ebc6d0b93ca807a40
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-boom
+    - r-assertthat >=0.2.0
+    - r-bsts >=0.9.0
+    - r-ggplot2
+    - r-zoo
+  run:
+    - r-base
+    - r-boom
+    - r-assertthat >=0.2.0
+    - r-bsts >=0.9.0
+    - r-ggplot2
+    - r-zoo
+
+test:
+  commands:
+    - $R -e "library('CausalImpact')"           # [not win]
+    - "\"%R%\" -e \"library('CausalImpact')\""  # [win]
+
+about:
+  home: https://google.github.io/CausalImpact/
```

Let's include the `dev_url`:

```suggestion
  home: https://google.github.io/CausalImpact/
  dev_url: https://github.com/google/CausalImpact
```